src/utils/logging.py

```diff
@@ -1,5 +1,3 @@
 from loguru import logger
 
-import functools
-
-logger.add("logs/file_1.log/", retention="100 days")
\ No newline at end of file
+logger.add("logs/file_1.log/", retention="100 days")
```

Comment:
Rename log file - file_1 was just a test
possibly remove retention, or use size instead of days.